Services/CO.CDP.OrganisationInformation.Persistence/AuthenticationKey.cs

```diff
@@ -13,6 +12,7 @@ public class AuthenticationKey : IEntityDate
 
     [ForeignKey(nameof(Organisation))]
     public int? OrganisationId { get; set; }
+    public bool? Revoked { get; set; }
```

Comment:
I'd suggest making it non-null, default `false`. This way we only have two states to deal with. What would `null` mean in this context anyway?

Reply:
Well created null because it was existing table.  Will make it default false.